src/styles/styled.d.ts

```
@@ -1,8 +1,7 @@
-import { theme } from "./Theme";
+import { theme } from './Theme'
 
-
-type Theme = typeof theme;
+type Theme = typeof theme
 
 declare module 'styled-components' {
-    export interface DefaultTheme extends Theme { }
+  export interface DefaultTheme extends Theme {}
```

ESLint

[error] An interface declaring no members is equivalent to its supertype.

(@typescript-eslint/no-empty-object-type)